clang/include/clang/Tooling/Inclusions/IncludeStyle.h

```diff
@@ -152,6 +152,10 @@ struct IncludeStyle {
   /// \version 10
   std::string IncludeIsMainSourceRegex;
 
+  /// When sorting includes in each block, ignore file extensions.
```

Comment:
I currently don't have a better name, but this isn't correct. You are not ignoring it, but still using it to break ties. Even if the name stays, I think there should be more explanation, maybe even an example.